azerothcore/azerothcore-wotlk
# refactor(Core/DB/SAI): Migrate Tanaris NPC's to SAI

## CHANGES PROPOSED:
-  Migrates Stone Watcher of Norgannon to SAI.
-  Migrates Steward of Time to SAI.
-  Migration of hardcoded text from core scripts to DB/SAI.

## ISSUES ADDRESSED:
- Closes 



## TESTS PERFORMED:
-
-



## HOW TO TEST THE CHANGES:

- ``.go cre 23289`` Talk to **Steward of Time** after completing the first quest see that the ``Please take me to the master  lair.`` menu will become available
- `.q add 2954` then add the creature` .npc add 7918` Stone Watcher of Norgannon
see that the gossip menu will go according to the c ++ script

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
